frameworks/cassandra/tests/config.py

```diff
@@ -89,6 +93,15 @@ def run_backup_and_restore(
     delete_data_job = get_delete_data_job(node_address=job_node_address)
     verify_deletion_job = get_verify_deletion_job(node_address=job_node_address)
 
+
+    # Ensure the keyspaces we will use aren't present.
+    try:
+        jobs.run_job(delete_data_job)
+    except:
```

Comment:
:+1: to the try-catch block though is there a more specific exception we can catch here?